misc/assets/icons.svg

```diff
@@ -0,0 +1,16 @@
+<svg version="1.1" viewBox="0 0 1000 200" xmlns="http://www.w3.org/2000/svg">
```

Comment:
@martinmacko47 Da sa to prepisat aj cez pomocny subor. Ked som si otvaral `attachments.svg` v prehliadaci (Mozilla) vsetko bolo OK. ALE github (aj gitlab) ma tento subor problem spravne nacitat. Takze by som to asi robil tak, aby bol kazdy `SVG` subor sam o sebe nezavisly.

Reply:
Github to svg posiela s headrom:
```
Content-Security-Policy: default-src 'none'; style-src 'unsafe-inline'; sandbox
```
Takze nic externe to neloadne. Cize nic nam neostava, len vsetky definicie mat vsade nakopirovane. Kazdopadne dodrzujme, ze veci, co maju rovnake ID v roznych suboroch su identicke.